module/input/SensorFilter/tests/SensorFilter.cpp

```diff
@@ -46,25 +46,25 @@ using utility::support::Expression;
 
 TEST_CASE("Test MotionModel Orientation", "[module][input][SensorFilter][MotionModel][orientation]") {
     // Create our motion model
-    UKF<float, MotionModel> filter{};
+    UKF<double, MotionModel> filter{};
```

Comment:
I thought you were changing from `double` to `float`, not `float` to `double`?

Reply:
The main modules UKF filter uses double because it has numerical issues with float but I have changed to float in the test.